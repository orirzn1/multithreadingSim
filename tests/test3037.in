L16
S8
O9
N6

T0
I@0x00000000
SUB $6, $0, $5
SUB $6, $5, $5
STORE $6, $7, $6
ADDI $1, $0, 0x4
SUB $1, $7, $6
SUB $0, $4, $1
LOAD $4, $4, $3
SUB $3, $4, $5
HALT $0

T1
I@0x00000000
SUBI $7, $4, 0x3
ADDI $0, $3, 0x2
ADDI $5, $7, 0x2
LOAD $7, $2, $7
STORE $3, $5, $1
LOAD $3, $5, 0x4
HALT $0

T2
I@0x00000000
SUB $3, $2, $5
ADDI $0, $4, 0x1
ADDI $7, $2, 0x2
ADD $7, $1, $7
STORE $2, $2, $2
SUBI $4, $0, 0x4
HALT $0

T3
I@0x00000000
ADD $3, $2, $7
STORE $3, $2, $0
ADDI $5, $0, 0x1
LOAD $0, $0, $5
SUBI $2, $2, 0x4
SUBI $5, $3, 0x4
HALT $0

T4
I@0x00000000
ADD $4, $4, $3
ADDI $1, $2, 0x4
SUB $6, $5, $2
SUBI $6, $2, 0x4
ADDI $2, $6, 0x4
ADDI $4, $2, 0x2
SUB $5, $3, $7
STORE $5, $5, $2
HALT $0

T5
I@0x00000000
SUB $2, $2, $4
STORE $4, $6, $7
LOAD $7, $4, $6
LOAD $2, $1, 0x2
STORE $3, $4, $3
STORE $7, $5, 0x1
LOAD $3, $5, 0x3
SUBI $5, $2, 0x2
ADDI $4, $5, 0x4
HALT $0

D@0x0
0x6
0x8
0x7
0x1
0x5
0x2
