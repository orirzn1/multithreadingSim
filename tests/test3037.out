
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x2	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x1	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFD	R5 = 0x0	R6 = 0x0	R7 = 0x2
Register file thread id 3:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0xFFFFFFFC	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFC	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x4	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x1	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0xFFFFFFFF
Blocked MT CPI for this program 4.897959

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x2	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x1	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFD	R5 = 0x0	R6 = 0x0	R7 = 0x2
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFC	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFC	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x4	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x1	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.877551

